python/cudf/cudf/core/column/lists.py

```diff
@@ -48,6 +50,24 @@ def offsets(self):
     def list(self, parent=None):
         return ListMethods(self, parent=parent)
 
+    def to_arrow(self):
+        offsets = self.offsets.to_arrow()
+        elements = self.elements.to_arrow()
+        pa_type = pa.list_(elements.type)
+
+        if len(elements) == elements.null_count:
+            elements = pa.NullArray.from_pandas([None] * len(elements))
```

Comment:
We should handle this before converting `elements.to_arrow()` above.

Reply:
fixed